lib/api/request/login_request.dart

```diff
@@ -0,0 +1,30 @@
+import 'package:json_annotation/json_annotation.dart';
+
+part 'login_request.g.dart';
+
+@JsonSerializable()
+class LoginRequest {
+  @JsonKey(name: 'grant_type')
+  String grantType;
+  @JsonKey(name: 'email')
+  String email;
+  @JsonKey(name: 'password')
+  String password;
+  @JsonKey(name: 'client_id')
+  String clientId;
+  @JsonKey(name: 'client_secret')
+  String clientSecret;
```

Comment:
I think you can follow this config to automatic convert to `snake case` for API request/response fields https://github.com/nimblehq/flutter_templates/issues/43

then we can get rid of these `JsonKey` config.
```suggestion
  String grantType;
  String email;
  String password;
  String clientId;
  String clientSecret;
```